tv-shows/src/fetchers/reviews.ts

```diff
@@ -0,0 +1,23 @@
+import { IReview } from '../typings/Review.type';
+import { fetcher } from './fetcher';
+import { mutator } from './mutators';
+
+interface IReviewsResponse {
+	reviews: Array<IReview>;
+}
+
+export function getManyReviews(url: string) {
+	return fetcher<IReviewsResponse>(url);
+}
+
+export function createReview(url: string, { arg }: { arg: any }) {
+	return mutator(url, { arg });
+}
+
+export function updateReview(url: string, { arg }: { arg: any }) {
+	return mutator(url, { arg }, 'PATCH');
```

Comment:
If you ask me the 'method' parameter should be a part of the second 'init' parameter, besides arg so that it mimics the `init` from fetch.